api/transport/propagation.go

```diff
@@ -37,6 +38,11 @@ type CreateOpenTracingSpan struct {
 	ExtraTags     opentracing.Tags
 }
 
+const (
+	//TracingTagStatusCode is the span tag key for the YAPRC status code.
+	TracingTagStatusCode = "rpc.yarpc.status_code"
```

Comment:
Don't status codes differ based on protocol? Does yarpc have a separate layer of abstraction for `status_codes`? 
If not, I would expect to see something like `rpc.grpc.status_code`

Reply:
Yes, yarpc does have: https://github.com/yarpc/yarpc-go/blob/dev/yarpcerrors/codes.go#L29

Reply:
Good to know - out of curiosity, where is the underlying grpc status_code added to span?

Reply:
The grpc status code is not added to spans in yaprc currently